public/js/admin.js

```diff
@@ -1,4 +1,7 @@
+/* global EmojiConvertor */
+
 async function getNotes() {
+    const emoji = new EmojiConvertor();
```

Comment:
Does this not need `emoji.init_env();` as used in `updateChores` of `whiteboard.js`?

Reply:
This was tested and works. I'm not sure that it needs it due to that.